flang/lib/Optimizer/Builder/Runtime/Main.cpp

```diff
@@ -16,13 +16,17 @@
 #include "flang/Optimizer/Dialect/FIRType.h"
 #include "flang/Runtime/main.h"
 #include "flang/Runtime/stop.h"
+#ifdef FLANG_CUDA_SUPPORT
+#include "flang/Runtime/CUDA/init.h"
+#endif
```

Comment:
What is the reason for protecting this include with an ifdef, if this header contains only a runtime signature, it does not seem like it require some CUDA support when building flang.

The `initCuda` dynamic flag seems enough to me to control the feature.

Reply:
The header is not present if the cuda fortran rubtime is not compiled

Reply:
Whether the build configuration compiles CUFRuntime or not, `init.h` is still present it the git checkout.